package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@civic/solana-gateway-react": "^0.4.6",
+    "@civic/solana-gateway-react": "^0.4.9",
```

Comment:
we can try using 0.4.10 because that release happened at the same time as the rust crate release we use